services/web-app/next.config.js

```diff
@@ -80,6 +80,16 @@ module.exports = withMDX({
         destination: `/assets/${library.host}/${library.org}/${library.repo}/${slug}/latest`
       })
 
+      rewrites.push({
+        source: `/assets/${slug}/assets`,
+        destination: `/assets/${library.host}/${library.org}/${library.repo}/${slug}/latest/assets`
+      })
+
+      rewrites.push({
+        source: `/assets/${slug}/:ref*/assets`,
+        destination: `/assets/${library.host}/${library.org}/${library.repo}/${slug}/:ref*/assets`
+      })
+
```

Comment:
Am I right that this would mean we'd have URLs like:
`next.carbondesignsystem.com/assets/some-thing/assets`?

If so, could we maybe chat about that to see if we're heading down a weird path?

Reply:
yeah so, a library's url is "/assets/library-slug" and this is a page for a library's assets, so I put it under "/assets/library-slug/assets" , I see how that's weird but it also makes sense to me to put the library's "assets" page under [library-url]/assets 🤔

Reply:
The url felt a little weird to me too when I was testing, but agree that unless naming of something changes, it makes sense. Tagging @aubrey-oneal and @kingtraceyj just as an FYI here.

Reply:
I'll create a separate issue for side nav (and URL) naming to not block this PR.

Reply:
@mattrosno We are trying to create a distinction between the UI Shell 'Assets' and the library assets. I think it's OK to get specific with what assets you are viewing. You aren't looking at all assets just the specific library ones. Using this logic we could go down the path of putting the word library in front of each nav item, but there's no real need since it's unrelated to the top nav item. We've had multiple people, seperately comment on that it's a little confusing to have both the top level nav item say assets as well as the side nav say assets. i think getting specific on this one category adds clarity.

Reply:
Updating the first item to be "Library assets" and the url to` /assets/carbon-charts/library-assets` is simple enough, and adds clarity.

Reply:
I'm okay with that for this PR. We're looking at side nav in a separate issue to ensure all of the other items, and their order, are right.

Reply:
fixed! new sample url: http://localhost:3000/assets/carbon-charts/library-assets

Reply:
Some notes to toss in the URL vs nav issue:

Can you refresh my memory about the root-most `/assets`? Are we saying that a Library is an Asset and also Libraries can *have* Assets, some of which might be other Libraries? That's what the current URL scheme implies, but I want to confirm that's the intent.

Is `/assets/button` ever a valid url? Or is it always `/assets/some-library-name/button`?

If it's only ever `/assets/some-library`, then I'd vote for being more explicit in the URL about the type of object you're viewing: `/libraries/carbon-charts`. Or even more explicitly: `/assets/libraries/carbon-charts/assets`.

At the moment, there's a mismatch between the breadcrumb and the URL (see screenshot). Assets, Libraries, and a specific library are referenced as navigation contexts on the page, but the URL is missing the indication of the specific library being nested under the set of all libraries.

If a library is not considered as an asset itself and is only a container of *other* assets, then I'd vote for `/libraries/carbon-charts/assets` as opposed to `/assets/libraries/carbon-charts/assets` so that the left nav becomes the more definitive source for the URL format.

(Marking as unresolved so the thread doesn't get lost until we copy/paste into an issue).

<img width="531" alt="image" src="https://user-images.githubusercontent.com/51208233/164272489-aea6aa79-116c-4366-948a-890880111b71.png">